model/fn-execution/src/main/proto/beam_fn_api.proto

```diff
@@ -311,13 +311,13 @@ message ProcessBundleProgressRequest {
   string instruction_reference = 1;
 }
 
-// A specification containing required set of fields and labels required
-// to be set on a MonitoringInfo for the specific URN for SDK->RunnerHarness
+// A specification containing required set of fields and labels required to be
+// set on a MonitoringInfo for a given URN-type, for SDK->RunnerHarness
```

Comment:
That's not correct. The specific URN has a set of required field.
ElementCount has a PCollection.
ExecutionTimes have a PTransform.

URN-type refers to the value type. counter int64, counter double, gauge string, gauge int64, etc.

Reply:
Yea, "type" was not what I meant here; context was [discussion w/ Robert](https://github.com/apache/beam/pull/7868#discussion_r257795396) about whether these specs describe "specific URNs" or "URN categories" (e.g. "user metrics").